### PR TITLE
Fix looking up multipart form temporary files

### PR DESCRIPTION
**Pull request by crazytonyli**
Fixes flaky unit tests.

---

- [x] Please check here if your pull request includes additional test coverage.
- [x] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

### CODE REVIEW

**Review by mokagio — APPROVED** [review]
Nice! Thank you.

File: Tests/WordPressKitTests/Tests/Utilities/URLSessionHelperTests.swift [hunk 1]
```diff
@@ -308,28 +308,24 @@ class URLSessionHelperTests: XCTestCase {
...
-            fm.enumerator(atPath: fm.temporaryDirectory.path)
-        ].compactMap { $0 }
-
-        var result: Set<String> = []
-        for enumerator in enumerators {
-            while let file = enumerator.nextObject() as? String {
-                result.insert(file)
+        let files = try fm.contentsOfDirectory(atPath: fm.temporaryDirectory.path)
+            .filter {
+                uuidRegex.numberOfMatches(in: $0, range: NSMakeRange(0, $0.count)) == 1
```
Comment (guarani) [inline Tests/WordPressKitTests/Tests/Utilities/URLSessionHelperTests.swift]: nitpick: this also works:
```suggestion
                UUID(uuidString: $0) != nil
```
and avoids the long regex.

**Review by guarani — APPROVED** [review]
Thanks a lot, @crazytonyli! Tested locally and working great 🙇

File: Tests/WordPressKitTests/Tests/Utilities/URLSessionHelperTests.swift (hunk 1 above)
Comment (crazytonyli) [reply Tests/WordPressKitTests/Tests/Utilities/URLSessionHelperTests.swift]: Thanks for suggestion! Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/781/commits/a7f92975139b6491519184474d7af586bec54557

**Pull request merged**